### ISSUE TITLE
PointGraphics initially fail to render in Columbus 2.5D mode

### ISSUE THREAD

**Issue by zglennie** [CesiumGS/cesium#8690]
## Summary

In 2.5D Columbus mode, Point entities are initially not visible, but become visible when the camera angle changes sufficiently.

## Sandcastle example:

This can be demonstrated with a minor change to the "[Drawing on Terrain](https://sandcastle.cesium.com/?src=Drawing%20on%20Terrain.html)" Sandcastle example.

Simply change the "pixelSize" in the createPoint function from 5 to 20:
```
function createPoint(worldPosition) {
    var point = viewer.entities.add({
        position : worldPosition,
        point : {
            color : Cesium.Color.WHITE,
            pixelSize : 20,
            heightReference: Cesium.HeightReference.CLAMP_TO_GROUND
        }
    });
    return point;
}
```

## Steps to Reproduce:

1. In the [Drawing on Terrain](https://sandcastle.cesium.com/?src=Drawing%20on%20Terrain.html) Sandcastle example, set a larger pixelSize so that it's easy to see the Point entities.
2. Run the modified Sandcastle example.
3. Use the scene mode picker to choose Columbus mode.
4. Draw a polyline. The PolylineGraphics will be shown, but not the extra-large PointGraphics vertices.
5. Zoom out or change the camera angle. The PointGraphics vertices will become visible.

## Actual/Undesired Behavior:

![points-not-rendering-columbus](https://user-images.githubusercontent.com/3003122/76878466-3f2d7100-684b-11ea-99bb-bd0c47230356.gif)

## Desired/Expected Behavior:

The Point entities should be visible upon creation, as they are in full 3D mode:

![points-rendering-properly-3D](https://user-images.githubusercontent.com/3003122/76878767-aba87000-684b-11ea-94b5-40264ddee10b.gif)

## Browser:

Chrome Version 80.0.3987.132
Firefox 74.0 (64-bit)

## Operating System:

Windows 10 version 1809 build 17763.1098

## Workarounds / things I've tried:

- I tried generating positions above the terrain surface (cartographic, with positive heights), and it did not help.

- I tried setting the `disableDepthTestDistance : Number.POSITIVE_INFINITY` and `distanceDisplayCondition : new Cesium.DistanceDisplayCondition()` which I believe should be the settings most conducive to always showing the Point entities, and it did not change the behavior.

- I introduced an automatic zoom-out-then-zoom-back in, by calling "move" methods on the Camera, and this does work, but requires waiting 50 milliseconds or so (the exact time varries), which is a jarring UX. In addition, the required move-back distance varies depending on the geometry (note, in the gif, one of the points fails to render even as the others *do* render, and does not render until we move back further.

If anyone has a fix or a good workaround, it would be very helpful to me!

**Comment by OmarShehata** [comment]
Thanks for reporting this @zglennie - this is definitely a bug. I think the reason that that disableDepthDistance etc aren't working is because the point isn't actually there. This [minimal Sandcastle example](https://sandcastle.cesium.com/#c=jVLRTuMwEPwVKy9Npcgp3NtdqCgBwUnctSoFXiIhY29bC8eu1ptwHOLfsZsWWpAQG0Xyzs7sbiZuBbJWwyMgO2IWHlkJXjc1v1ljaU+u09JZEtoC9jLGnivLQhAgBmyCrtUqyH9upRJBENw6NGrWcdJ+ZV/6vyobnythlRSeDHCh1Mw5cy/wpCFyNu2NlGITpy2FQfPGStIB7W9HdotysKRJg4/6dFOKsXJeR8H7KqVACidhf/A5uvoUFgjg04NBxuI76Ge76jA2SHcaxpDOuJ2PK2PKby9+z86yfeJK/wNzpf9DIB8OPhSXoBdLmsIcEKyEt3YX+zgvL0d/Jnez8d35dHz99/S9yUt3XLu4sfJLI0sDAr9lIkLtWhgZk256J1lSeHoyMOwEx7peOSTWoEk5zwnqlQk/2Of3jXwA4tL7qCzyrahQumVaHVXJh9tTJUwa4X2ozBuzNqtKhkUe+Hsy44TSdjFuAY14ipTlwfCyAznnRR7SzyrqHAj0uExXfQU) shows what I think is going on:

![point](https://user-images.githubusercontent.com/1711126/76991835-a2d59e00-6920-11ea-8e4c-205e4a63781a.gif)

it seems to not get clamped to terrain until the camera moves to a specific distance. @lilleyse do you know what might be going on?

**Comment by lilleyse** [comment]
Yeah seems like an edge case related to changing the scene mode and not updating clamped objects until a new terrain tile is loaded.

**Comment by ebogo1** [comment]
I think this [forum post](https://community.cesium.com/t/bug-primitive-label-columbus-view-relativetoground/14003) is the same bug with labels in CV.
